set.mm

```diff
@@ -648630,67 +648746,47 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
-    ltexp1dd $p |- ( ph -> ( A ^ N ) < ( B ^ N ) ) $=
-      ( clt wbr cexp co ltexp1d mpbid ) ABCIJBDKLCDKLIJHABCDEFGMN $.
-  $}
-
-  $( ~ zsqrtelqelz generalized to positive integer roots.  (Contributed by
-     Steven Nguyen, 6-Apr-2023.) $)
-  zrtelqelz $p |- ( ( A e. ZZ /\ N e. NN /\ ( A ^c ( 1 / N ) ) e. QQ ) ->
-                    ( A ^c ( 1 / N ) ) e. ZZ ) $=
-    ( cz wcel cn c1 cdiv co ccxp cdenom cfv wceq 3ad2ant3 cexp 3ad2ant1 syl2anc
-    cq syl wb qden1elz w3a qdencl nnrpd crp 1rp a1i nnzd nnne0d 1exp cc cxproot
-    simp2 zcn fveq2d ibir eqtrd cn0 simp3 nnnn0d denexp 3eqtr2rd exp11d mpbid
-    zq ) ACDZBEDZAFBGHIHZQDZUAZVGJKZFLZVGCDZVIVJFBVIVJVHVEVJEDVFVGUBMUCFUDDVIUE
-    UFVIBVEVFVHULZUGZVIBVMUHVIFBNHZFVGBNHZJKZVJBNHZVIBCDVOFLVNBUIRVIVQAJKZFVIVP
-    AJVIAUJDZVFVPALVEVFVTVHAUMOVMABUKPUNVEVFVSFLZVHVEWAVEAQDWAVESAVDATRUOOUPVIV
-    HBUQDVQVRLVEVFVHURVIBVMUSVGBUTPVAVBVHVEVKVLSVFVGTMVC $.
+    cgcd wceq wi nnnn0 dvdsexpim syl3an gcdnncl simpl1 simpl3 expgcd syl3an3 wb
+    crp simp1 3ad2ant3 simp2 gcdeq syl2anc biimpar eqtrd exp11nnd simprd syl2an
+    gcddvds eqbrtrrd ex impbid ) ADEZBDEZCDEZFZABGHZACIJZBCIJZGHZVFAKEZVGBKEZVH
+    CLEZVJVMUAAMZBMZCUBZABCUCUDVIVMVJVIVMNZABSJZABGVTWAACVIWAUKEZVMVFVGWBVHVFVG
+    NWAABUEOPQVTAVFVGVHVMUFOVFVGVHVMUGVTWACIJZVKVLSJZVKVIWCWDTZVMVHVFVGVPWEVSAB
+    CUHUIQVIWDVKTZVMVIVKDEVLDEWFVMUJVIACVFVGVHULVHVFVPVGVSUMZRVIBCVFVGVHUNWGRVK
+    VLUOUPUQURUSVIWABGHZVMVFVGWHVHVFVNVOWHVGVQVRVNVONWAAGHWHABVBUTVAPQVCVDVE $.
+
+  $( ~ dvdsexpnn generalized to include zero bases.  (Contributed by SN,
+     15-Sep-2024.) $)
+  dvdsexpnn0 $p |- ( ( A e. NN0 /\ B e. NN0 /\ N e. NN ) ->
+                                    ( A || B <-> ( A ^ N ) || ( B ^ N ) ) ) $=
+    ( cn0 wcel cn cdvds wbr cexp co wb cc0 wceq wo elnn0 wa adantl syl bibi12d
+    cz wi dvdsexpnn 3expia cc nncn expeq0 sylan 0exp breq1d nnexpcl sylan2 nnzd
+    nnnn0 0dvds bitrd adantr 3bitr4rd breq1 oveq1 syl5ibr expdimp breqtrrd 2thd
+    nnz dvds0 breq2 breq2d syl5ibrcom impancom bicomd breq12 simpl oveq1d simpr
+    breq12d ccase syl2anb 3impia ) ADEZBDEZCFEZABGHZACIJZBCIJZGHZKZVSAFEZALMZNB
+    FEZBLMZNWAWFUAZVTAOBOWGWIWHWJWKWGWIWAWFABCUBUCWHWIWAWFWIWAPZWFWHLBGHZLCIJZW
+    DGHZKWLWDLMZWJWOWMWIBUDEWAWPWJKBUEBCUFUGWLWOLWDGHZWPWLWNLWDGWAWNLMZWICUHZQU
+    IWLWDTEWQWPKWLWDWAWICDEZWDFECUMZBCUJUKULWDUNRUOWIWMWJKZWAWIBTEXBBVDBUNRUPUQ
+    WHWBWMWEWOALBGURWHWCWNWDGALCIUSUISUTVAWGWAWJWFWGWAPZWFWJALGHZWCWNGHZKXCXDXE
+    WGXDWAWGATEXDAVDAVERUPXCWCLWNGXCWCTEWCLGHXCWCWAWGWTWCFEXAACUJUKULWCVERWAWRW
+    GWSQVBVCWJWBXDWEXEBLAGVFWJWDWNWCGBLCIUSVGSVHVIWAWFWHWJPZLLGHZWNWNGHZKWAXHXG
+    WAWNLWNLGWSWSVOVJXFWBXGWEXHALBLGVKXFWCWNWDWNGXFALCIWHWJVLVMXFBLCIWHWJVNVMVO
+    SUTVPVQVR $.
+
+  $( ~ dvdssq generalized to positive integer exponents.  (Contributed by SN,
+     15-Sep-2024.) $)
+  dvdsexpb $p |- ( ( A e. ZZ /\ B e. ZZ /\ N e. NN ) ->
+                                    ( A || B <-> ( A ^ N ) || ( B ^ N ) ) ) $=
```

Comment:
This should probably go to Main in a future PR.

Reply:
> This should probably go to Main in a future PR.
I agree, but then most of the previous theorems must also be moved, since this theorem depnds on them.

Reply:
Indeed: it would drag with it all the results used in its proof.